### DISCUSSION TITLE
2.8 Breaking Changes Expected?

### DISCUSSION THREAD

**Discussion by freshleafmedia** [area17/twill#1581] [Q&A] [answered]
Hello,

Is it expected that the pending 2.8 release contains breaking changes?

I have noticed that Capsule migrations are no longer recognised as of 2.8.0-beta.2 due to the path being changed.
Before it was `app/Twill/Capsules/<CAPSULE_NAME>/database/migrations` now it's `app/Twill/Capsules/<CAPSULE_NAME>/Database/migrations`. Note the capitalisation of `Database`

I believe this is the offending line: https://github.com/area17/twill/compare/2.8.0-beta.1...2.8.0-beta.2#diff-ee5de863e3c0587659268d0b3156e3ecfcdcb9f27b5c98bf99099ec590d0f871R183

**Comment by haringsrob** [comment, marked as answer]
Ah good catch, I guess I never noticed because I am on mac. I will make a fix for this.

**Reply by freshleafmedia** [reply]
Ah, that classic case-insensitivity bug :+1:
